modules/7_ibmcloud/variables.tf

```diff
@@ -24,6 +24,7 @@ variable "cluster_domain" {
 variable "cluster_id" {
   default = "test-ocp"
 }
+variable "name_prefix" {}
```

Comment:
Do we want to use the zone in load_balancer name?
I understand this is VPC but to identify the cluster using this LB is in which zone may help.
@bpradipt

Reply:
@yussufsh ok

Reply:
zone info added for LB and security group.